GetFitLA_v3/app/src/main/res/layout/maps.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
```

Comment:
Again, use a Relative Layout

Reply:
We've decided to not allow for a rotated screen since it makes our UI look ugly. That being said you've brought up a good point about consistency which I will address right now in an updated code